src/test/java/org/mule/extension/ValidateArbitraryPrecisionTestCase.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Seems like a good time to create the first MTF test for the module :)

Reply:
Not today